src/tests/certify.rs

```diff
@@ -674,3 +674,154 @@ fn mock_trust_flow_all() {
 
     insta::assert_snapshot!(result);
 }
+
+#[test]
+fn mock_trust_flow_all_allow_multiple() {
```

Comment:
Maybe one more test which touches the code path which suggests the user add `--allow-multiple-publishers` would be good (it could be identical to this test but without the flag).

Reply:
The test above (`mock_test_flow_all`) does hit the codepath which adds the `--allow-multiple-publishers` code path. You don't see it in the captured output because the message is logged with `warn!` instead of to the `out`, so it doesn't appear in the test output unfortunately.

I considered looking into adding a new mechanism to register a `BasicTestOutput` as a handler for the tracing subscriber, but it's a bit tricky to do well (as the subscriber is shared between multiple test runs on multiple threads, happening concurrently - so connecting it back to the correct test is a bit sketchy). I don't know what the best way to have warnings like that be captured in the test output would be unfortunately. (probably would need to be TLS and bound to the current thread's warning logs only?)